### DISCUSSION TITLE
Kong 2.5.1 not yet on docker hub

### DISCUSSION THREAD

**Discussion by dsteinkopf** [Kong/kong#7847] [Help] [answered]
Is it "expected" that the version 2.5.1 (commited 7 days ago) is not yet available on docker hub?
Thank you very much.

**Comment by guanlan** [comment, marked as answer]
@dsteinkopf We are working on the publishing.

**Reply by dsteinkopf** [reply]
Thank you. Now it is available. Successfully updated my systems this night.
